ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantStateWriter.scala

```diff
@@ -52,7 +53,8 @@ class KeyValueParticipantStateWriter(writer: LedgerWriter, metrics: Metrics) ext
         sourceDescription.getOrElse(""),
         writer.participantId,
       )
-    commit(submissionId, submission)
+    // TODO(KVL-874): stop using NoOp
```

Comment:
Please don't add TODOs in the code; keep it to the issue tracker.

Reply:
Sure, these were just markers for the next steps in this task.